src/lib/parquet/src/Flow/Parquet/stubs.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
I think I would move that file to `src/lib/parquet/src` since it's not under any namespace and `functions.php` reflects Flow\Parquet namespace

Reply:
I put it there initially, but then I saw `functions.php` and thought it was better to put them together :)) I'll move it around, no worries!